mbs_results/data_cleaning.py

```diff
@@ -182,14 +182,63 @@ def load_manual_constructions(
     )
 
 
+def join_manual_constructions(
+    df: pd.DataFrame,
+    manual_constructions: pd.DataFrame,
+    reference: str,
+    period: str,
+    **config
+):
+    """
+    function to change datatypes of columns based on config file
```

Comment:
The description needs updating , also I think saying "function"  is redundant

Reply:
Updated in newest commit